.azure/vs2022.yml

```diff
@@ -1302,114 +1325,153 @@ stages:
-               -D ArgumentParser_DIR=$(Agent.BuildDirectory)/swift-argument-parser/cmake/modules
-               -D LLBuild_DIR=$(Agent.BuildDirectory)/swift-llbuild/cmake/modules
-               -D SwiftCollections_DIR=$(Agent.BuildDirectory)/swift-collections/cmake/modules
-               -D SwiftCrypto_DIR=$(Agent.BuildDirectory)/swift-crypto/cmake/modules
-               -D SwiftDriver_DIR=$(Agent.BuildDirectory)/swift-driver/cmake/modules
-               -D SwiftSystem_DIR=$(Agent.BuildDirectory)/swift-system/cmake/modules
-               -D TSC_DIR=$(Agent.BuildDirectory)/swift-tools-support-core/cmake/modules
-               -D Yams_DIR=$(Agent.BuildDirectory)/Yams/cmake/modules
-               -D SQLite3_LIBRARY=$(Pipeline.Workspace)/sqlite-$(arch)-3.36.0/Library/sqlite-3.36.0/usr/lib/SQLite3.lib
-               -D SQLite3_INCLUDE_DIR=$(Pipeline.Workspace)/sqlite-$(arch)-3.36.0/Library/sqlite-3.36.0/usr/include
+              cmakeArgs:
+                -B $(Agent.BuildDirectory)/swift-asn1
+                -D BUILD_SHARED_LIBS=NO
+                -D CMAKE_BUILD_TYPE=Release
+                -D CMAKE_MT=mt
+                -D CMAKE_Swift_COMPILER=$(workspace.root)/toolchain-amd64/Library/Developer/Toolchains/unknown-Asserts-development.xctoolchain/usr/bin/swiftc.exe
+                -D CMAKE_Swift_COMPILER_TARGET=$(platform)-unknown-windows-msvc
+                -D CMAKE_Swift_FLAGS="-resource-dir $(Pipeline.Workspace)/windows-sdk-$(arch)/Library/Developer/Platforms/Windows.platform/Developer/SDKs/Windows.sdk/usr/lib/swift -L$(Pipeline.Workspace)/windows-sdk-$(arch)/Library/Developer/Platforms/Windows.platform/Developer/SDKs/Windows.sdk/usr/lib/swift/windows"
+                -G Ninja
+                -S $(Build.SourcesDirectory)/swift-asn1
+                -D dispatch_DIR=$(Agent.BuildDirectory)/libdispatch/cmake/modules
+                -D Foundation_DIR=$(Agent.BuildDirectory)/foundation/cmake/modules
+          - task: CMake@1
+            inputs:
+              cmakeArgs:
+                --build $(Agent.BuildDirectory)/swift-asn1
+          - task: CMake@1
+            inputs:
+              cmakeArgs:
+                -B $(Agent.BuildDirectory)/swift-certificates
+                -D BUILD_SHARED_LIBS=NO
+                -D CMAKE_BUILD_TYPE=Release
+                -D CMAKE_MT=mt
+                -D CMAKE_Swift_COMPILER=$(workspace.root)/toolchain-amd64/Library/Developer/Toolchains/unknown-Asserts-development.xctoolchain/usr/bin/swiftc.exe
+                -D CMAKE_Swift_COMPILER_TARGET=$(platform)-unknown-windows-msvc
+                -D CMAKE_Swift_FLAGS="-resource-dir $(Pipeline.Workspace)/windows-sdk-$(arch)/Library/Developer/Platforms/Windows.platform/Developer/SDKs/Windows.sdk/usr/lib/swift -L$(Pipeline.Workspace)/windows-sdk-$(arch)/Library/Developer/Platforms/Windows.platform/Developer/SDKs/Windows.sdk/usr/lib/swift/windows"
+                -G Ninja
+                -S $(Build.SourcesDirectory)/swift-package-manager
```

Comment:
This looks wrong. Shouldn't it be `$(Build.SourcesDirectory)/swift-certificates`?

Reply:
Ugh!  Yes.